pdns/dnsdist.hh

```diff
@@ -298,7 +298,10 @@ struct DNSDistStats
     {"fd-usage", getOpenFileDescriptors},
     {"dyn-blocked", &dynBlocked},
     {"dyn-block-nmg-size", [](const std::string&) { return g_dynblockNMG.getLocal()->size(); }},
-    {"security-status", &securityStatus}
+    {"security-status", &securityStatus},
+    // Latency histogram
+    {"latency-sum", &latencySum},
+    {"latency-count", &responses}, // This is the same metric as responses but uses the metric name as expected by Prometheus
```

Comment:
Adding these values here would cause them to be sent over carbon and the HTTP API as well, where they have no meaning. Perhaps it's time to replace the pair in this vector with a proper structure so we can specify that these are for prometheus use only? This is not really a blocker since carbon and API consumers could simply discard them, but it feels a bit wrong.

Reply:
Well if Prometheus can use them to generate histograms other consumers might be able to use those stats as well

Reply:
Fair enough. I would bet they are never going to get used outside of prometheus but that's fine, it's probably not worth the additional code needed to hide them.